server/sonar-server/src/main/java/org/sonar/server/user/ws/GroupsAction.java

```diff
@@ -61,74 +63,55 @@ public void define(NewController context) {
       .setDescription("Lists the groups a user belongs to. Requires Administer System permission.")
       .setHandler(this)
       .setResponseExample(getClass().getResource("groups-example.json"))
+      .addSelectionModeParam()
+      .addSearchQuery("users", "group names")
+      .addPagingParams(25)
+      .setChangelog(new Change("6.4", "Paging response fields as been moved under a 'paging' node"))
```

Comment:
Paging response fields are moved to a Paging object
Paging response fields moved to a Paging object